<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMindMapDtLineTool</class>
 <widget class="CDiagramTool" name="CMindMapDtLineTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QToolButton" name="m_btnPath">
     <property name="toolTip">
      <string>Change Edge Style</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/mac/res.qrc">
       <normaloff>:/res/icons/node-edge.png</normaloff>:/res/icons/node-edge.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnLineWeight">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Change Line Weight</string>
     </property>
     <property name="text">
      <string>Border Weight</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/mac/res.qrc">
       <normaloff>:/res/icons/global-border-weight.png</normaloff>:/res/icons/global-border-weight.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnLineStyle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Change Line Style</string>
     </property>
     <property name="text">
      <string>Border Style</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/mac/res.qrc">
       <normaloff>:/res/icons/global-border-style.png</normaloff>:/res/icons/global-border-style.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_btnLineColor">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Change Line Color</string>
     </property>
     <property name="text">
      <string>Pen Color</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/mac/res.qrc">
       <normaloff>:/res/icons/global-border-color.png</normaloff>:/res/icons/global-border-color.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CDiagramTool</class>
   <extends>QWidget</extends>
   <header>core/CDiagramWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
